meetings/2019-01-14.md

```diff
@@ -0,0 +1,142 @@
+
+One thing we talked about as an alternative is that the votes belong to the projects. This has the benefit of making it clearer that anyone can participate. 
+
+Mdawson: how would that work for non-top-level projects. 
+
+Myles: potentially they would not have a vote at the CPC. 
+
+Jory & Ben: favor having an explicit list.
+
+Joe: I’ll concur that I think being project based will add some complexities. 
+
+Myles: Seems like people prefer the clarity, Is 2 members the appropriate amount from each project?
+
+Mdawson: Or we can do up to 2?
+
+Discussion of tradeoffs between having one and two delegates. 
+
+Andy: people need to come into the conversation with the same expectations/understanding of what the roles/responsibilities are. 
+
+Myles/Mdawson: how much of this has to be in the charters vs the governance?
+
+Andy: there’s a draft that’s been prepared. You can take those as a guideline for what’s appropriate. As a generality I prefer to but it in the charter vs the bylaws. Classically the must have in the charter is scope, who can participate, etc. Most of them have a charter and nothing more. But you can have two separate documents. 
+
+Mdawson: we can get agreement on scope, then split it across two documents and that shoudl be fine right?
+
+Myles: I agree, keep it as streamlined as possible. My preference is to have 1 voting member per top level project. And to have regular members and observers.
+
+Tierney: I can see what’s happened in Node happening here, where we get people who become members but then have to drop off a lot. To me being a member doesn’t necessarily mean they are going to be engaged. Having everyone participate as an observer maybe recognizes that trend,
+
+Andy: the one good thing about goverenance is that the same issues show up everywhere. As a generality you are safer looking at other organizations that have thrivign individual participation than you are trying to invent your own. 
+
+Sarah: I can add a little from the CNCF. it has ind. Steering committess for different projects and the CPC is focused on the inbound projects. They also have contributors to this group, 9 seats which are voting seats, and everyone actually votes even those with non-binding votes.
+Anyone can attend the meeting, and there are people who are TOC contributors and you do a certain amount of work to earn that. 
+
+MDawson: that would be close to what we have written with the members vs. regular members. If we made additional reqs it would be closest.
+
+Group: likes the non-binding vote idea. Anyone can make a non-binding vote. 
+
+Myles: if people are up for it, I think we should merge the two PRs & open a new PR with alternative language to flesh out the ideas we were discussing here, and aim to land something next week?
```

Comment:
Nit: flush out (I guess)

Reply:
@thefourtheye flesh is correct in this case.
https://www.merriam-webster.com/words-at-play/top-10-commonly-confused-words-vol-2

Reply:
Ah, thanks @joesepi. I have never seen this usage before.